abci/types/application.go

```diff
@@ -95,6 +97,14 @@ func (BaseApplication) ApplySnapshotChunk(req RequestApplySnapshotChunk) Respons
 	return ResponseApplySnapshotChunk{}
 }
 
+func (BaseApplication) PrepareProposal(req RequestPrepareProposal) ResponsePrepareProposal {
+	return ResponsePrepareProposal{BlockData: req.BlockData}
```

Comment:
[This comment](https://github.com/celestiaorg/celestia-core/pull/898/files#r1037958275) also applies here